float32 pressure
float32 altitude
float32 temperature
uint64 timestamp
uint64 error_count
